SUPPORTED_LANGUAGES.md

```diff
@@ -141,6 +141,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | Mercury                 | mercury                |         |
 | MIPS Assembler          | mips, mipsasm          |         |
 | Mint                    | mint                   | [highlightjs-mint](https://github.com/mint-lang/highlightjs-mint) |
+| Mirth                   | mirth                  | [highlightjs-mirth](https://github.com/casuallyblue/highlightjs-mirth) |
```

Comment:
You're wanting us to host the repo though, right? I'm confused.

Reply:
Ah yeah, I wasn't sure of the turnaround time on either so I thought I should go ahead and request to be added to the list while the other issue was in progress, sorry.

Reply:
@joshgoebel would it be preferable to use the highlightjs hosted repo in the PR already even though it doesn't exist currently?

Reply:
Yep. Then we'll make it all happen.

Reply:
It should be fixed now